.github/workflows/test.yml

```diff
@@ -74,6 +74,21 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: Setup dev environment
+        run: |
+          pip install -r requirements-dev.txt
+          ### pytest-sentry configuration ###
+          if [ "$GITHUB_REPOSITORY" = "getsentry/self-hosted" ]; then
+            echo "PYTEST_SENTRY_DSN=https://5a620019b5124cbba230a9e62db9b825@o1.ingest.us.sentry.io/6627632" >> $GITHUB_ENV
+            echo "PYTEST_SENTRY_TRACES_SAMPLE_RATE=0" >> $GITHUB_ENV
```

Comment:
I would use a different dsn if you haven't already -- and ~slightly hide it in github variables (doesn't have to be a secret, just a repository variable) -- should help with forks sending garbage into this

Reply:
This DSN is specific for self-hosted tests so it won't be shared with sentry. I can put it as a variable in GH